docs/projects/ad7606x_fmc/index.rst

```diff
@@ -0,0 +1,438 @@
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`ad7616x_fmc HDL project <projects/ad7606x_fmc>`
+-  :ref:`SPI_ENGINE <spi_engine>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - SYNC_BITS
+     - :git-hdl:`library/util_cdc/sync_bits.v <library/util_cdc/sync_bits.v>` **
+     - ---
+   * - AD_EDGE_DETECT
+     - :git-hdl:`library/common/ad_edge_detect.v <library/common/ad_edge_detect.v>`
+     - ---
+   * - AXI_AD7606x
+     - :git-hdl:`library/axi_ad7606x <library/axi_ad7606x>` *
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_ad7606x>`
+   * - AXI_CLKGEN
+     - :git-hdl:`library/axi_dmac <library/axi_clkgen>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_clkgen>`
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac <library/axi_dmac>`
+     - :ref:`here <axi_dmac>`
+   * - AXI_HDMI_TX
+     - :git-hdl:`library/axi_hdmi_tx <library/axi_hdmi_tx>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_hdmi_tx>`
+   * - AXI_I2S_ADI
+     - :git-hdl:`library/axi_i2s_adi <library/axi_i2s_adi>`
+     - ---
+   * - AXI_PWM_GEN
+     - :git-hdl:`library/axi_pwm_gen <library/axi_pwm_gen>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_pwm_gen>`
+   * - AXI_SPDIF_TX
+     - :git-hdl:`library/axi_spdif_tx <library/axi_spdif_tx>`
+     - ---
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid <library/axi_sysid>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine <library/spi_engine/axi_spi_engine>`  **
+     - :ref:`here <spi_engine axi>`
+   * - SPI_ENGINE_EXECUTION
+     - :git-hdl:`library/spi_engine/spi_engine_execution <library/spi_engine/spi_engine_execution>` **
+     - :ref:`here <spi_engine execution>`
+   * - SPI_ENGINE_INTERCONNECT
+     - :git-hdl:`library/spi_engine/spi_engine_interconnect <library/spi_engine/spi_engine_interconnect>` **
+     - :ref:`here <spi_engine interconnect>`
+   * - SPI_ENGINE_OFFLOAD
+     - :git-hdl:`library/spi_engine/spi_engine_offload <library/spi_engine/spi_engine_offload>` **
+     - :ref:`here <spi_engine offload>`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom <library/sysid_rom>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - UTIL_I2C-MIXER
+     - :git-hdl:`library/util_i2c_mixer <library/util_i2c_mixer>`
+     - ---
+   * - UTIL_CPACK2
+     - :git-hdl:`library/util_pack/util_cpack2 <library/util_pack/util_cpack2>` *
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/util_cpack>`  
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for INTF=0 (parallel interface)
+   -   ``**`` instantiated only for INTF=1 (serial interface)
+
+Software related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-no-os:`AD7606 - No-OS Driver <drivers/adc/ad7606>`
```

Comment:
Can you rephrase this line, like in this example: https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad7616_sdz/index.rst?plain=1#L381 ?

Reply:
Done.